testsuite/tck/src/test/java/io/smallrye/openapi/tck/DeploymentProcessor.java

```diff
@@ -47,12 +48,21 @@
 import io.smallrye.openapi.runtime.io.Format;
 import io.smallrye.openapi.runtime.scanner.FilteredIndexView;
 
-public class DeploymentProcessor implements ApplicationArchiveProcessor {
+public class DeploymentProcessor {
     private static Logger LOGGER = Logger.getLogger(DeploymentProcessor.class.getName());
     public static volatile ClassLoader classLoader;
 
-    @Override
-    public void process(Archive<?> archive, TestClass testClass) {
+    public void observeDeployment(@Observes final BeforeDeploy beforeDeploy) {
+        DeploymentDescription deployment = beforeDeploy.getDeployment();
+        Archive<?> testableArchive = deployment.getTestableArchive();
+        if (testableArchive != null) {
+            process(testableArchive);
```

Comment:
This can presumably be removed with RC3, correct?

Reply:
No, there are still a few testable archives which test the model creation APIs.

Reply:
For reference it's [OASFactoryErrorTest](https://github.com/eclipse/microprofile-open-api/blob/master/tck/src/main/java/org/eclipse/microprofile/openapi/tck/OASFactoryErrorTest.java) and [ModelConstructionTest](https://github.com/eclipse/microprofile-open-api/blob/master/tck/src/main/java/org/eclipse/microprofile/openapi/tck/ModelConstructionTest.java)